tests/backend/test_supabase_integration.py

```
@@ -0,0 +1,620 @@
+    """Supabase と FastAPI の統合テスト"""
+
+    @pytest.mark.asyncio
+    async def test_api_channels_with_supabase(self):
+        """
+        SupabaseバックエンドでのAPI チャンネル一覧取得テスト
+
+        Test: Supabaseをバックエンドとして使用してもチャンネル一覧APIが正常動作する
+        """
+        # Act & Assert
+        with pytest.raises(ImportError):
+                from src.backend.supabase_main import app
```

Ruff (0.11.9)

`src.backend.supabase_main.app` imported but unused

Remove unused import: `src.backend.supabase_main.app`

(F401)